### PR TITLE
Changes for the next Mayavi release which uses VTK's new pipeline.

### PR DESCRIPTION
**Pull request by dmsurti**
1. Bump up to version 5.0.0.
2. Document all enhancements and fixes.
3. Add self name as contributor.

### CODE REVIEW

File: docs/CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,44 @@
+Mayavi 5.0.0
+============
+
+Enhancements
+------------
+
+22 Dec 2014 (PR)
```
Comment (prabhuramachandran) [inline docs/CHANGES.txt]: Thanks but, these changes are by DS and not PR. :)

File: mayavi/__init__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
     Part of the Mayavi project of the Enthought Tool Suite.
 """
 
-__version__ = '4.3.2'
+__version__ = '5.0.0'
```
Comment (prabhuramachandran) [inline mayavi/__init__.py]: Why not 4.4?

Comment (dmsurti) [inline mayavi/__init__.py]: As we upgrade from VTK 5.10.1 to VTK 6, especially using the new pipeline architecture, I thought this qualified for major version upgrade.

Comment (GaelVaroquaux) [inline mayavi/__init__.py]: Hey Deepak,

Thanks for doing all this! It's really great to know that Mayavi
development is alive and kicking! This software is really widely used.

File: docs/CHANGES.txt [hunk 3]
```diff
@@ -1,3 +1,44 @@
+Mayavi 5.0.0
```
Comment (prabhuramachandran) [inline docs/CHANGES.txt]: This needs to be changed.

**Pull request merged**